internal/cmd/table.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	courseUser "github.com/edulinq/autograder/internal/api/courses/users"
+	"github.com/edulinq/autograder/internal/api/users"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func ListServerUsersTable(response core.APIResponse) string {
+	var responseContent users.ListResponse
+	util.MustJSONFromString(util.MustToJSON(response.Content), &responseContent)
```

Comment:
!
Unmarked Must function.

I don't like having these conversions in these functions.
Back when these functions were in their respective CMDs, then these conversions make sense (we know exactly what the types should be).
But here, you are letting anyone call these with any response types
(and thus can cause a panic on the conversions).

I think you should be sending in the users instead of the responses.